crawlerr/server.py

```
@@ -283,45 +309,61 @@ async def get_server_info(ctx: Context) -> Dict[str, Any]:
+            logger.info("📚 Vector collection ready: %s", c_status)
         
-        logger.info("Crawlerr server started successfully ✓")
+        logger.info("[bold green]🎉 Crawlerr server started successfully![/bold green]")
         
+    except ToolError as e:
+        logger.exception("[red]❌ Critical startup error: %s[/red]", e)
+        logger.info("[dim]🤷 Server started but some services may be unavailable[/dim]")
+    except (ConnectionError, TimeoutError) as e:
+        logger.warning("[yellow]⚠️  Service connection failed: %s[/yellow]", e)
+        logger.info("[dim]🤷 Server started but some services may be unavailable[/dim]")
     except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)